frontend/static/css/style.css

```diff
@@ -5,53 +5,33 @@ html, body {
 }
 
 body {
-  background-color: #f5f5f5;
-  font-family: "Roboto", Helvetica, Tahoma, sans-serif;
-}
-
-.header{
-  font-size: 20px;
-  color: rgb(66, 66, 66);
-  letter-spacing: 0.02em;
-  display:flex;
-  align-items: center;
-  height: 64px;
-  padding: 0 16px 0 40px;
-
-  font-family: "Roboto", Helvetica, Tahoma, sans-serif;
+  background-color: #7f7f7f;
+  font: 14px/1.5 -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Helvetica, Arial, sans-serif;
```

Comment:
Could you prioritize Roboto?

Reply:
This is the de facto font system stack. We use it also in Bootstrap.

See https://css-tricks.com/snippets/css/system-font-stack/

Reply:
Alright, that actually looks pretty reasonable. Thanks for the suggestion!